plugin/audit_null/audit_null.cc

```diff
@@ -175,6 +183,10 @@ static MYSQL_THDVAR_INT(event_order_check_exact, PLUGIN_VAR_RQCMDARG,
                         "Plugin checks exact event order.", NULL, NULL, 1, 0, 1,
                         0);
 
+static MYSQL_THDVAR_INT(extended_log, PLUGIN_VAR_RQCMDARG,
```

Comment:
It's better to declare this var as `GLOBAL`-only (`MYSQL_SYSVAR_INT`) since we are using a single global buffer for the output anyway.

Reply:
But doesn't that make thread level control useful? If it's a threadvar, you can turn it on for only one thread, and other threads won't accidentally modify the status variables, breaking tests.

Reply:
And is audit_null used outside MTR testing, in production? This is only present in that plugin, that's why I didn't see it as an issue. I can make it debug only, but that will also make all related tests debug only.

Reply:
It's not used in production, but if it's used as a template for other audit plugins, it might run the risk of it getting out. Let's make it debug-only and having the tests be debug-only would be fine.